src/style/pinny.scss

```diff
@@ -83,11 +83,14 @@
 // 1. `display: flex` is coming from `.pinny__wrapper`
 // 2. This allows the content to grow or shrink as necessary based on content
 //    and the header/footer
+// 3. Fills content background after rotation on Android 4.3 (HTC, Samsung Galaxy S3)
 
 .pinny__content {
     overflow: auto;
     flex: 1; // 1, 2
 
+    background: white; // 3
+
     -webkit-overflow-scrolling: touch;
 }
```

Comment:
@scalvert Pinny content background is a pinny issue. And it's fixed with this line. But I see that Shade is independent plugin and its fix should be done separately. 

I've also spoke to @Helen-Mobify and following her words it's a different issue then https://github.com/mobify/pinny/pull/66

So whats the plan in this case? Reopen this PR for Pinny fix only and create another in the Shade?